pypit/flatfield.py

```diff
@@ -73,6 +77,8 @@ class FlatField(processimages.ProcessImages, masterframe.MasterFrame):
       Number of knots in the spectral dimension
 
     """
+
+    # TODO user settings need to be added to these codes!!
```

Comment:
No, I doubt that is the approach we will be taking.

Reply:
Again let's discuss.